manifests/ca.pp

```diff
@@ -100,13 +100,13 @@
   $ssl_key_size = 1024,
   $ca_expire    = 3650,
   $key_expire   = 3650,
-  $key_cn       = '',
-  $key_name     = '',
-  $key_ou       = '',
+  $key_cn       = undef,
+  $key_name     = undef,
+  $key_ou       = undef,
   $tls_auth     = false,
 ) {
 
-  include openvpn
+  include ::openvpn
```

Comment:
With puppet4+, the absolute path check is no longer needed, and in fact, `include foo` is preferred over `include ::foo`